client/extensions/woocommerce/components/button/style.scss

```diff
@@ -0,0 +1,367 @@
+/** @format */
```

Comment:
We need this file because we want just the styling of WooCommerce theme, but using wordpress/component's button?

Reply:
nvm i see it. :D

Reply:
:D Yes, we are keeping the styling of our buttons but using WP components. We are not making changes on the UI unless needed.